src/foam/u2/view/FObjectView.js

```diff
@@ -97,6 +103,12 @@ foam.CLASS({
       name: 'dataWasProvided_',
       documentation: 'Set to true if data was initially provided. Used to implement classIsFinal.'
     },
+    {
+      class: 'Boolean',
+      name: 'selectByDefault',
+      documentation: 'If true, the choiceView will select the first element',
+      value: 'true'
```

Comment:
Make make the default value be:
expression: function(placeholder) { return ! placeholder; }
Or, get rid of this property entirely and just have it always select the first by default if placeholder isn't set.